agent/util-scripts/unittests

```diff
@@ -203,6 +205,8 @@ declare -A expected_status=(
     [test-11]=4
     [test-12]=1
     [test-13]=2
+    # XXX - this is wrong: it needs iostat-postprocess in order to return 0
```

Comment:
It means that I forced the unit test to pass even though the solution is not quite correct. It should exit with status 0, but it cannot find iostat-postprocess, so it exits with status 1. I did not have the stamina to pursue it yesterday. I'll take another look today.